Automate/RedHatConsulting_Satellite6/Infrastructure/VM/Retirement/StateMachines/VMRetirement.class/__class__.yaml

```diff
@@ -333,11 +333,32 @@ object:
       max_time: 
   - field:
       aetype: state
-      name: DeleteFromVMDB
+      name: UpdateLDAP
```

Comment:
This was specifically not `LDAP` so that if someone was using some other `VMDB` then it was agnostic. We could update to `UpdateOrDeleteFromVMDB` or just `UpdateVMDB` since technically a delete would be an update.

Reply:
Okay i see now this was just a shift down, the point of the existing `DeleteFromVMDB` was for a person to say update or delete from LDAP (if using LDAP as a VMDB) during that step.